### ISSUE TITLE
SEO optimisation 

### ISSUE THREAD

**Issue by dlupu** [kundigo/konfido-landing-page-drivers#6]
[Search for konfido in google](https://www.google.fr/search?q=konfido) our website does not even appear on it ! WTF?
![konfido google search](https://cloud.githubusercontent.com/assets/247936/19213198/1da541d0-8d65-11e6-9daa-694151726bea.png)

**Comment by franzejr** [comment]
Yeah. I think only `Konfido` in the entire World, it will be difficult. But if you search for konfido taxi, konfido uber or konfido chauffeur, you will find it!
